src/__screenshot_tests__/form-select-screenshot-test.tsx

```diff
@@ -2,30 +2,22 @@ import {openStoryPage, screen} from '../test-utils';
 
 import type {Device} from '../test-utils';
 
-const devices: Array<[Device, string?]> = [
-    ['DESKTOP', undefined],
-    ['MOBILE_IOS', 'iPhone'],
-];
+const devices: Array<Device> = ['DESKTOP', 'MOBILE_IOS'];
 
-test.each(devices)(
-    'Select elements on the initial state appear properly on %s',
-    async (device, userAgent) => {
-        const page = await openStoryPage({
-            id: 'components-forms-select--default',
-            device,
-            userAgent,
```

Comment:
unrelated. This was a TS error. I think CI doesn't check TS on test files

Reply:
hmm, I added a `yarn ts-check` in the `ci` target... it should check types in jenkins (at least in master)